src/app-layout/classic.tsx

```diff
@@ -158,7 +138,6 @@ const ClassicAppLayout = React.forwardRef(
     } = useFocusControl(!!activeDrawerId, true, activeDrawerId);
 
     const onNavigationToggle = useStableCallback((open: boolean) => {
-      setNavigationOpen(open);
```

Comment:
This line is removed here now, because the line below `fireNonCancelableEvent(onNavigationChange, { open })` now updates all connected states, controlled and uncontrolled